packages/uswds-core/src/styles/mixins/helpers/at-media.scss

```diff
@@ -19,6 +21,19 @@
     @media all and (min-width: #{$bp}) {
       @content;
     }
+  } @else if map.has-key($custom-breakpoints, $quoted-bp) {
+    @if unit(map.get($custom-breakpoints, $quoted-bp)) ==
+      "rem" or
+      unit(map.get($custom-breakpoints, $quoted-bp)) ==
+      "px"
+    {
+      $bp: map.get($custom-breakpoints, $quoted-bp);
+      @media all and (min-width: #{$bp}) {
+        @content;
+      }
+    } @else {
+      @warn 'Custom breakpoint `#{$quoted-bp}` has the unit #{unit(map.get($custom-breakpoints, $quoted-bp))}. Custom breakpoints must be either rem or px.';
```

Comment:
Good point. We should update anyways because the test instructions in description say non-px values will trigger a warning. So we can remove the `either rem or px` note and just say `px`.

Reply:
Whoops, I realize my comment was made a lil while ago and is a bit outdated. It looks like it doesn't reference `rem` in the warning message anymore. I'll go ahead and make the update about the breakpoint not generating.

Reply:
Added in [f319030](https://github.com/uswds/uswds/pull/6048/commits/f31903066e5a572c4978d2102c5b138ac3e02fc4)